docs/Developers/Testing_Bears.rst

```diff
@@ -124,9 +124,9 @@ passed.
             self.check_results(
                 self.uut,
                 file,
-                Result.from_values("TooManyLinesBear",
-                                   "Too many lines"
-                                   settings={'max_number_of_lines': int=20}))
+                [Result.from_values('TooManyLinesBear',
+                                   'Too many lines',
+                                   settings={'max_number_of_lines': int=20})])
```

Comment:
The beginning opening single quotes on line 127, 128 and `s` of `settings` should be aligned.